lib/workers/repository/process/index.js

```diff
@@ -23,6 +23,11 @@ async function processRepo(config) {
           const [, type, branchName] = check.match(new RegExp(checkMatch));
           config.masterIssueChecks[branchName] = type;
         });
+      }
+      const checkAllMatch = ' - \\[x\\] <!-- rebase-all-open-prs -->';
+      const checkedRebaseAll = issue.body.match(new RegExp(checkAllMatch, 'g'));
+      if (checkedRebaseAll && checkedRebaseAll.length) {
```

Comment:
I think the length check is redundant - it should be `null` if there are no matches